cookbook/routing/multiple_routers.rst

```diff
@@ -0,0 +1,19 @@
+.. index::
+   single: Routing; Multiple Routers
+
+Using the Symfony CMF ChainRouter to combine multiple routers
+=============================================================
+
+The Symfony CMF ChainRouter allows to use more than one router. A main
+use case is to keep the :doc:`default symfony routing system </book/routing>`
+available when writing a custom controller.
+
+.. caution::
+
+    If you simply need a way to load routes determined in a different
+    way, using :doc:`a custom route loader <custom_route_loader>` is
+    simpler than writing your own controller.
+    Writing a custom controller is justified when the routes can not be
+    statically determined.
+
+TODO: a bit of code examples?
```

Comment:
i need to wrap this up, if we agree that this PR is going in the right direction.